elfo-network/src/discovery/mod.rs

```diff
@@ -27,20 +30,26 @@ mod diff;
 /// TODO: should be different for groups and actors.
 const INITIAL_WINDOW_SIZE: i32 = 100_000;
 
+#[message]
+struct MakeConnections;
```

Comment:
(minor, naming)

https://actoromicon.rs/ch03-03-sources.html#delays

> adhering to best practice by using the 'Tick' suffix

Also, `Make` does not obviously mean "Open"